### PR TITLE
Removal of GenotypeReader from fingerprintChecker.

### PR DESCRIPTION
**Pull request by kasont**

### CODE REVIEW

File: src/main/java/picard/fingerprint/FingerprintChecker.java [hunk 1]
```diff
@@ -132,11 +128,13 @@ public void setpLossofHet(final double pLossofHet) {
      */
     public Map<String,Fingerprint> loadFingerprints(final File fingerprintFile, final String specificSample) {
         final IntervalList loci = this.haplotypes.getIntervalList();
-        final GenotypeReader reader = new GenotypeReader();
-        final GenotypeReader.VariantIterator iterator = reader.read(fingerprintFile, loci);
+        final VCFFileReader read = new VCFFileReader(fingerprintFile);
+        final CloseableIterator<VariantContext> iterator = read.iterator();
+        //final GenotypeReader reader = new GenotypeReader();
```
Comment (tfenne) [inline src/main/java/picard/fingerprint/FingerprintChecker.java]: Please remove commented out lines of code.

File: src/main/java/picard/fingerprint/FingerprintChecker.java [hunk 2]
```diff
@@ -132,11 +128,13 @@ public void setpLossofHet(final double pLossofHet) {
...
+        //final GenotypeReader.VariantIterator iterator = reader.read(fingerprintFile, loci);
 
         SequenceUtil.assertSequenceDictionariesEqual(this.haplotypes.getHeader().getSequenceDictionary(),
-                                                     iterator.getSequenceDictionary());
+                read.getSequenceDictionary(fingerprintFile));
```
Comment (tfenne) [inline src/main/java/picard/fingerprint/FingerprintChecker.java]: align under `this.hap....`

File: src/main/java/picard/fingerprint/FingerprintChecker.java [hunk 3]
```diff
@@ -132,11 +128,13 @@ public void setpLossofHet(final double pLossofHet) {
      */
     public Map<String,Fingerprint> loadFingerprints(final File fingerprintFile, final String specificSample) {
         final IntervalList loci = this.haplotypes.getIntervalList();
-        final GenotypeReader reader = new GenotypeReader();
-        final GenotypeReader.VariantIterator iterator = reader.read(fingerprintFile, loci);
+        final VCFFileReader read = new VCFFileReader(fingerprintFile);
```
Comment (tfenne) [inline src/main/java/picard/fingerprint/FingerprintChecker.java]: `read` is not a good name for an object which _reads_ variants.  I suggested `reader`, `vcfReader`, `in`, etc.

**Comment by tfenne** [comment]
Since GenotypeReader is only used in FingerprintChecker why not remove the deprecated class from Picard?  I doubt anyone else is actually using it.

**Comment by kasont** [comment]
@tfenne thank you for the comments.

**Comment by yfarjoun** [comment]
no need to close..yo can push up a new commit and it will update the PR.

**Comment by kasont** [comment]
Thank you. @yfarjoun

**Pull request closed without merging**